Assets/Scripts/Entity/Enemy/Boss/DevilBoss.cs

```diff
@@ -51,6 +52,7 @@ private void Awake()
         player = GameObject.FindGameObjectWithTag("Player");
         animator = self.model.GetComponent<Animator>();
         phaseTwo = gameObject.GetComponent<PhaseTwo>();
+        DeathAudio = GetComponents<AudioSource>()[1];
```

Comment:
This may not be causing any problems now, but it's highly bug prone. You should never rely on the order of the components in a prefab. If we move the audio sources around relative to each other, then this will no longer be the correct audio source. Instead, since you've already serialized the field above, all you need to do is just drag the right audio component from the boss prefab to the slot.

Reply:
Also I can barely hear the audio, looks like you set the volume to be quite low.

Reply:
Duly noted. And yes volume is meant to be low. In particular, because of the effect I want to achieve (namely, increasing volume as the player falls into oblivion).